### PR TITLE
Create ZPA Epic template and extend Definition of Done in PR template

### PR DESCRIPTION
**Pull request by remcowesterhoud**
## Description

As discussed in our team meeting, it is convenient to create issues in their respective repositories when we make a change that impacts them. This removes an extra step where they need to ask for information which we can provide them.

We discussed the options of creating an Epic template and adding it here, but not all changes will be made through an Epic. As an alternative we discussed adding it to the Definition of Done in our PR template.

Our conclusion was to add it in both places. This is what this PR achieves.

## Related issues

closes #

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [x] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (koevskinikola) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:63]: Found a typo here:

```suggestion
<!-- The Epic might have an impact on other teams within Camunda. We should create an issue for
```

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (koevskinikola) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:64]: I assume its:
```suggestion
them, explaining the changes they will have to make. -->
```

**Comment by koevskinikola** [comment]
:thought_balloon: @remcowesterhoud should we also include a link to the Optimize issue tracker? For some topics, they might be involved as well ([ex](https://github.com/camunda/product-hub/issues/618#issuecomment-1323878608)).

**Comment by remcowesterhoud** [comment]
Yes we should, can't believe I missed those 😅

**Review by koevskinikola — APPROVED** [review]
:+1: Looks great!

:thought_balloon: I'm not sure if my review is enough, or if another team member should have a look since I haven't handled a lot of ZPA epics. Feel free to merge it if you don't think more reviews are necessary.

**Review by saig0 — APPROVED** [review]
@remcowesterhoud awesome. :rocket:

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (korthout) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:22]: Do we need this part in our EPICs?

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (korthout) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:31]:
```suggestion
## Concept
<!-- Provide a high-level explanation of the implementation concept. For example, when a new BPMN element gets added,
explain how it is supposed to work. -->
```

Previously, I've used the concept to explain the implementation concept rather than the concept of the feature itself (i.e. from the user's perspective).

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (korthout) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:75]: 🔧 I'd prefer to also add a section for `Decisions` or `Discussions`, where we explain the answers to resolved questions, e.g. specific design decisions, or answers to questions that don't make sense to mention in the rest of the breakdown.

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (korthout) [inline .github/ISSUE_TEMPLATE/zpa_epic.md:5]:
```suggestion
title: '[EPIC] '
```

❓ Not sure if we would want to do this, but so far, our epic issues are often prefixed with something like this. What do you think?

**Review by korthout — APPROVED** [review]
Looks great @remcowesterhoud Thanks for this! 🚀 

🔧 I have a few questions, and suggestions. Please have a look before merging.

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (remcowesterhoud) [reply .github/ISSUE_TEMPLATE/zpa_epic.md:22]: I copied the feature request template, forgot to take this out 😄

File: .github/ISSUE_TEMPLATE/zpa_epic.md
Comment (remcowesterhoud) [reply .github/ISSUE_TEMPLATE/zpa_epic.md:75]: I feel like this could be the same category as `Open Question`. Maybe we should rename it to `Discussions and Decisions`. An ongoing discussions/decision is IMO the same as an open question. WDYT?

Comment (korthout) [reply .github/ISSUE_TEMPLATE/zpa_epic.md:61]: That's a good point, and it might be easier to use because it avoids us having to move things around. 

👿 There's a counter-argument to be made that it's easier to find what is still unclear, or what has been decided in the split variant.

👍 I'm fine with it either way

**Comment by remcowesterhoud** [comment]
bors merge

**Comment by remcowesterhoud** [comment]
bors retry

**Comment by remcowesterhoud** [comment]
bors cancel

**Comment by remcowesterhoud** [comment]
bors merge

**Comment by remcowesterhoud** [comment]
Same issues as before with TCC, let's try again.

bors retry

**Pull request merged**
